app/auth/token.js

```diff
@@ -0,0 +1,30 @@
+const jwt = require('jsonwebtoken');
+
+/**
+ * validates the token, and return the user json.
+ * @param token
+ * @returns {Promise<user json>}
+ */
+async function validateToken(token){
+  return jwt.verify(token, process.env.SECRETKEY, async function(err, authData) {
+        if (err) {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443778)
```suggestion
    if(err) {
```